client/src/scssStyles/_mixins.scss

```diff
@@ -5,7 +5,7 @@
     }
   }
   @if $breakpoint == mobile {
-    @media only screen and (max-width: 599px) {
+    @media only screen and (max-width: 767px) {
```

Comment:
there was some weird formatting (it wasn't following the css for desktop or mobile) when the width was between 767-600px , so I just changed this for now...! @jameskokoska or @Cali0707 you might want to take a look at this?

Reply:
Yep no problem
I think it's because you would need to implement tablet styles before mobile would kick in
@Cali0707 I was thinking we just set them to the same, only design for mobile and desktop should be okay for now
(we can just keep them at the same value)